frontend/src/components/Collections/components/Grid/components/Row/DatasetRow/index.tsx

```diff
@@ -154,11 +155,12 @@ const DatasetRow: FC<Props> = ({
             checked={selected === dataset.id}
           />
           <div>{name}</div>
-          {!isLoading && (
-            <ErrorTooltip isFailed={isFailed} error={error} type={type} />
-          )}
+          {!isLoading &&
+            dataset.collection_access_type === ACCESS_TYPE.WRITE && (
```

Comment:
I think we can remove this change given that conditions needed to publish a private collection

Reply:
in the future when we share a private collection will we want users to see the errors?